### ISSUE TITLE
4.3.2+1 crashes on OS X El Capitan (10.11.6) upon opening after installation

### ISSUE THREAD

**Issue by Khord** [Yubico/yubioath-desktop#232]
Right after installation of 4.3.2+1 on my Mac (OSX 10.11.6), the app crashes immediately upon opening, showing the following in the send to Apple problem report (excerpt):

```
Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Dyld Error Message:
  Symbol not found: _getentropy
  Referenced from: /Applications/Yubico Authenticator.app/Contents/MacOS/../Frameworks/Python.framework/Versions/3.6/Python (which was built for Mac OS X 10.13)
  Expected in: /usr/lib/libSystem.B.dylib
```

Going back and downgrading, 4.2.0 is the latest version that launches for me, after going down the list and uninstalling / installing them one by one.

**Comment by Khord** [comment]
I ended up caving in and upgrading to High Sierra, and it started working after. Feel free to close this if it's not worth investigating at this point.

**Comment by dagheyman** [comment]
Thanks for the report! Glad that it started working after the upgrade, but we'll need to investigate if the new releases crashes on El Capitan.

**Comment by dagheyman** [comment]
Confirmed not crashing on Sierra, will try to reproduce on El Capitan later.

**Comment by rafassdotcom** [comment]
I can reproduce this in OSX 10.11.6, exactly the same.
`Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Dyld Error Message:
  Symbol not found: _getentropy
  Referenced from: /Applications/Yubico Authenticator.app/Contents/MacOS/../Frameworks/Python.framework/Versions/3.6/Python (which was built for Mac OS X 10.13)
  Expected in: /usr/lib/libSystem.B.dylib`
the latest version of the app that is working fine is the 4.2.0

**Comment by rafassdotcom** [comment]
I can reproduce this in OSX 10.11.6, exactly the same.

> `Crashed Thread: 0 Dispatch queue: com.apple.main-thread
Exception Type: EXC_BREAKPOINT (SIGTRAP)
Exception Codes: 0x0000000000000002, 0x0000000000000000
Exception Note: EXC_CORPSE_NOTIFY
Dyld Error Message:
Symbol not found: _getentropy
Referenced from: /Applications/Yubico Authenticator.app/Contents/MacOS/../Frameworks/Python.framework/Versions/3.6/Python (which was built for Mac OS X 10.13)
Expected in: /usr/lib/libSystem.B.dylib`

the latest version of the app that is working fine is the 4.2.0

**Comment by dagheyman** [comment]
There is a developer build here that you may test on your El Capitan system: https://yubico-builds.s3-eu-west-1.amazonaws.com/yubioath-desktop/yubioath-desktop-travis-el-capitan.app.tar

Note that the app is unsigned and doesn't have an installer, it's just meant for testing.

**Comment by vaurora** [comment]
I'm still getting this crash on El Capitan (10.11.6) using 4.3.5. Is the fix for this bug in that build? 4.2.0 works fine.

Process:               yubioath-desktop [78762]
Path:                  /Applications/Yubico Authenticator.app/Contents/MacOS/yubioath-desktop
Identifier:            com.yubico.yubioath
Version:               4.3 (4.3.5)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           yubioath-desktop [78762]
User ID:               502

Date/Time:             2019-05-05 13:48:31.721 -0700
OS Version:            Mac OS X 10.11.6 (15G22010)
Report Version:        11
Anonymous UUID:        D8AB10E6-1E2F-8765-29DC-0E1D85189284

Sleep/Wake UUID:       D5CDB9E0-5905-4306-8642-0765A554C0DA

Time Awake Since Boot: 980000 seconds
Time Since Wake:       5000 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Dyld Error Message:
  Symbol not found: _getentropy
  Referenced from: /Applications/Yubico Authenticator.app/Contents/MacOS/../Frameworks/Python.framework/Versions/3.6/Python
  Expected in: /usr/lib/libSystem.B.dylib

**Comment by dagheyman** [comment]
@vaurora Thanks for the report. Note that Apple no longer provides security updates for El Capitan, so my first advise would be to upgrade to a newer macOS version and see if that helps with the issue.
